src/main/java/com/minecolonies/blockout/controls/Button.java

```diff
@@ -6,6 +6,10 @@
 import net.minecraft.init.SoundEvents;
 import org.jetbrains.annotations.NotNull;
 
+import javax.swing.Timer;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
```

Comment:
Java awt stuff is quite often not well seen in these projects, isn't there a java.util Timer as well?